publisher-sdk/src/main/java/com/criteo/publisher/BidListener.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ *    Copyright 2020 Criteo
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.criteo.publisher
+
+import androidx.annotation.Keep
+
+/**
+ * Interface to be used in `App Bidding` integration mode to know when bidding has completed,
+ * before the Ad can be loaded.
+ *
+ * @see: https://publisherdocs.criteotilt.com/app/android/app-bidding/
+ */
+@Keep
+interface BidListener {
```

Comment:
`BidListener` is very generic and could also fit for InHouse. Do you think it might be good to insert the name of the integration to tight the listener ?
Like `AppBiddingListener` and `InHouseListener` ?

This is just a proposition, do not hesitate to confirm that you really want a `BidListener` for AppBidding.

Reply:
InHouse will actually have a different API. For InHouse it is necessary to return a `BidResponse` object in the callback param. So I was thinking: `BidListener` or `BidCompleteListener` for AppBidding and `BidResponseListener` for InHouse. WDYT?

Reply:
For InHouse it is necessary to return a BidResponse object in the callback param. So I was thinking: BidListener or `BidCompleteListener` for AppBidding and `BidResponseListener` for InHouse. I think the name of the listener should follow its usage, not the integration mode. WDYT?

Reply:
Ok then. Those names are precise enough to not lock us with a generic name.

Reply:
I will update the design doc with the API to get buy-in from all stakeholders.